clients/js/test/removePlugin.test.ts

```diff
@@ -754,3 +756,120 @@ test('it cannot use an invalid collection to remove a plugin on an asset', async
     },
   });
 });
+
+test('it cannot remove an authority managed plugin when the authority is None', async (t) => {
+  const umi = await createUmi();
+
+  const assetAuth = generateSigner(umi);
+  const asset = await createAsset(umi, {
+    updateAuthority: assetAuth,
+    plugins: [
+      pluginAuthorityPair({
+        type: 'Attributes',
+        data: { attributeList: [{ key: 'key', value: 'value' }] },
+        authority: nonePluginAuthority(),
+      }),
+    ],
+  });
+
+  await assertAsset(t, umi, {
+    ...DEFAULT_ASSET,
+    asset: asset.publicKey,
+    owner: umi.identity.publicKey,
+    updateAuthority: { type: 'Address', address: assetAuth.publicKey },
+    attributes: {
+      authority: {
+        type: 'None',
+      },
+      attributeList: [{ key: 'key', value: 'value' }],
+    },
+  });
+
+  const result = removePluginV1(umi, {
+    asset: asset.publicKey,
+    authority: assetAuth,
+    pluginType: PluginType.Attributes,
+  }).sendAndConfirm(umi);
+
+  await t.throwsAsync(result, { name: 'InvalidAuthority' });
+});
+
+test('it cannot remove an owner managed plugin when the authority is None', async (t) => {
+  const umi = await createUmi();
+
+  const assetAuth = generateSigner(umi);
+  const asset = await createAsset(umi, {
+    updateAuthority: assetAuth,
+    plugins: [
+      pluginAuthorityPair({
+        type: 'TransferDelegate',
+        authority: nonePluginAuthority(),
+      }),
+    ],
+  });
+
+  await assertAsset(t, umi, {
+    ...DEFAULT_ASSET,
+    asset: asset.publicKey,
+    owner: umi.identity.publicKey,
+    updateAuthority: { type: 'Address', address: assetAuth.publicKey },
+    transferDelegate: {
+      authority: {
+        type: 'None',
+      },
+    },
+  });
+
+  const result = removePluginV1(umi, {
+    asset: asset.publicKey,
+    pluginType: PluginType.TransferDelegate,
+  }).sendAndConfirm(umi);
+
+  await t.throwsAsync(result, { name: 'InvalidAuthority' });
+});
+
+test('it can remove an owner managed plugin from an asset the authority is None after transferring', async (t) => {
```

Comment:
Ok so just for business logic completeness, should we have one more test showing that you 'still' cannot remove an authority-managed plugin even after a transfer? 

I know the code doesn't work like that now but it would prevent regression if the revoke logic was being moved around.